docs/kb/function-tutorial-intro.md

```diff
@@ -0,0 +1,330 @@
+cd test
+git clone https://github.com/vmware-samples/vcenter-event-broker-appliance.git
+```
+
+### Install Docker
+
+First, you will need to sign up for a Docker account here: [DockerHub sign up](https://hub.docker.com/signup).  This will be required later when you want to push your customized Docker image to the DockerHub registry.  The DockerHub registry is an online library of container images.  Kubernetes clusters pull container images from registries.  Once you login for the first time, you can choose the "Personal", or free plan.
+
+Next, download the Docker dmg from: [Docker download](https://hub.docker.com/editions/community/docker-ce-desktop-mac/) and install.
+
+Now login using the Docker account you just signed up with (replace dduck username and VMware1! password with your own account info below).  Open a Terminal window and submit:
+```
+docker login --username dduck --password VMware1!
+```
+
+### Install and Configure `kubectl`
+
+`kubectl` is the command line utility used to access a Kubernetes cluster.  With `kubectl`, you can deploy the VEBA function in the form of Kubernetes secrets and pods, check logs, and troubleshoot the health of the Kubernetes cluster and VEBA application.  The `kubectl` command line utility determines which Kubernetes cluster to access and which certificate to use for authentication by reading a config file.  The default config file resides at: `~/.kube/config` on the VEBA appliance.
+
+Install `kubectl` with brew:
+
+```
+brew install kubectl
+kubectl version
+```
+
+Next we will copy the `.kube/config` file from the VEBA appliance to the local workstation.  You will need to know the IP address and root password of the deployed VEBA appliance and also verify that SSH has been enabled.  These parameters will have been set during the VEBA appliance deployment.  Once the config file is copied locally, we will export it so that it becomes the default `kubectl` config file.  You can switch which Kubernetes cluster `kubectl` points at by changing the `export KUBECONFIG` file link.  This example creates a "veba" folder to store the config file in and uses `scp` to copy the file from the VEBA appliance.  Replace <VEBA IP> below with the VEBA appliance IP address.  You will be prompted for the VEBA appliance root password.
+
+
+```
+cd ~
+mkdir veba
+cd veba
+scp 'root@<VEBA IP>:/root/.kube/config' .
+export KUBECONFIG=$HOME/veba/config
+```
+The above sample code is _**one**_ way of configuring the Kubernetes config file to use for access to the cluster.  How to navigate between multiple configs is documented here: [Kubernetes configure access](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/).
+
+
+Check that everything is working.
+- `kubectl config view`: returns the currently loaded config
+- `kubectl cluster-info`: returns info from the Kubernetes cluster (within the VEBA appliance)
+- `kubectl get ns`: returns the namespaces from the Kubernetes cluster
+```
```

Comment:
I think you need to add a newline here since the code block is intended in the web view.

Reply:
added line and removed the tutorial-kubectl-port.png as it was not needed.